Sources/GoodNetworking/CodableExtensions.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Codable.swift
+//  DepoSwiftExtensions
```

Comment:
I'd fix the file headers (in all files) considering this is now a public library